system_test/helpers/aest_nodes.erl

```diff
@@ -41,6 +41,68 @@
 -define(NODE_TEARDOWN_TIMEOUT, 0).
 -define(DEFAULT_HTTP_TIMEOUT, 3000).
 
+%% AWK script to keep only error, critical, alert and emergency log lines with
+%% all the extra lines following the log lines.
+%% In addition, it filter out the error lines caused by eper/watchdog.
+%% Update when https://www.pivotaltracker.com/story/show/157215253 is fixed.
```

Comment:
Could you please ensure the update of these scripts is tracked in Pivotal?

Reply:
Should the scripts even cater to existing bugs? Shouldn't we just highlight the bugs with the tests instead?

Reply:
If they don't it would just sit around until these bugs are fixed... Or do you suggest to merge something in master that would make system tests fail randomly?

Reply:
I would suggest that, to highlight the problems and to avoid having to remember to fix it later when the bugs are fixed. But I don't have a strong opinion on it 🙂